hilti/toolchain/include/ast/expressions/builtin-function.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2020-2021 by the Zeek Project. See LICENSE for details.
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <hilti/ast/declarations/parameter.h>
+#include <hilti/ast/expression.h>
+#include <hilti/ast/node.h>
+#include <hilti/ast/type.h>
+
+namespace hilti {
+namespace expression {
+
+/** AST node representing a builtin function. */
+class BuiltinFunction : public NodeBase, public trait::isExpression {
+public:
+    BuiltinFunction(std::string name, std::string cxxname, hilti::Type type,
```

Comment:
This could use a comment string explaining the semantics of the parameters, not immediately clear without further context.